openapi.yml

```diff
@@ -443,7 +443,7 @@ components:
           nullable: true
           format: int32
           description: "QoS Egress Queue ID"
-          minimum: 0
+          minimum: -1 # It can be used to set `queue_id` to default
```

Comment:
@Alopalao can you move the comment to the description, what's what the description is for, just so it can also be rendered nicely.

Reply:
Description moved.